conans/client/toolchain/__init__.py

```diff
@@ -1 +1,2 @@
 
+from .make import MakeToolchain
```

Comment:
Is this import necessary? We don't use this in the codebase

Reply:
It seemed it was necessary at the time. I will try to remove and see if it still works.